docs/usage.rst

```diff
@@ -58,7 +58,7 @@ Example:
 record-data
 %%%%
 
-Connects to a device and records ExG and orientation data into two separate files. Note that in CSV mode there will be an extra file for the marker events. In EDF mode, the data is actually recorded in BDF+ format (in 24-bit resolution).
+Connects to a device and records ExG and orientation data into two separate files. In EDF mode, the data is actually recorded in BDF+ format (in 24-bit resolution). Note that in CSV mode there will be two extra file for the marker events and recording metadata.
```

Comment:
"Note that in CSV mode there will be two extra file for the marker events and recording metadata." -> 

"Note that in CSV mode there will be two extra files. One for the marker events, and one for the metadata."